meta.yml

```diff
@@ -48,21 +48,20 @@ license:
 
 supported_coq_versions:
   text: 8.8 or later (use releases for other Coq versions)
-  opam: '{(>= "8.8" & < "8.11~") | (= "dev")}'
+  opam: '{(>= "8.8" & < "8.12~") | (= "dev")}'
```

Comment:
The README and opam files still claim compatibility with Coq 8.8-8.9, but you removed testing for these versions. Is there a reason for this?

Reply:
CI has had quite a lot of problems lately with GitHub fetching and so on. Tracking many CI builds for old versions can be a chore in that case. I guess I can add the other versions back if/when I haven't seen a lot of flaky builds for a while.

Reply:
OK, please feel free to ping me when you see such flaky builds. That's something that I'm willing to work on (even though until my defense, I'll have little time to allocate to that).